File: Fundamentals of C programming/step05/5.2/5.2.05.c
/*

Результатом операции десятичного сдвига вправо для целого числа 127127 будет число 1212. Написать программу, которая осуществляет операцию десятичного сдвига вправо для целого трёхзначного числа kk.

Входные данные:
Целое трёхзначное число kk.

Выходные данные:
Число, полученное из числа kk десятичным сдвигом вправо.

Sample Input:

850

Sample Output:

85

*/

#include <stdio.h>

int main() {
    int x; 
	scanf("%d", &x);
	printf("%d\n", x/10);
    
  return 0;
}

File: Python_Samura1 part 1/step05/5.02/5.2.05.py
n = int(input())
if n%5==0 and n%3==0:
    print('КрастиКрабс')
elif n%3 == 0:
    print('Красти')
elif n%5 == 0:
    print('Крабс')
else:
    print(n)
